internal/manifests/osbuild.go

```diff
@@ -119,6 +129,45 @@ func (o *OSBuildCreator) Create(ctx context.Context, osBuildConfig *osbuildv1alp
 	return nil
 }
 
+func mergeRepositories(osBuildConfigSpecDetails *osbuildv1alpha1.BuildDetails) error {
+	repos, err := getDefaultRepositories(osBuildConfigSpecDetails.Distribution, osBuildConfigSpecDetails.TargetImage.Architecture)
+	if err != nil {
+		return nil
+	}
+
+	if osBuildConfigSpecDetails.TargetImage.Repositories != nil {
+		repos = append(repos, *osBuildConfigSpecDetails.TargetImage.Repositories...)
```

Comment:
what about the case that would be a conclusion between the default repos to the customize repos?